src/components/ui/FileBrowser/FileList.tsx

```diff
@@ -41,25 +45,72 @@ export default function FileList({
   hideDotFiles,
   handleRightClick
 }: FileListProps): JSX.Element {
+  const { currentNavigationPath, getFileFetchPath, fetchFileAsJson } = useFileBrowserContext();
+  const { currentFileSharePath } = useZoneBrowserContext();
+
   const displayFiles = React.useMemo(() => {
     return hideDotFiles
       ? files.filter(file => !file.name.startsWith('.'))
       : files;
   }, [files, hideDotFiles]);
 
-  const hasZattrs = files.some(file => file.name === '.zattrs');
+  const [hasMultiscales, setHasMultiscales] = React.useState(false);
+  const [thumbnailSrc, setThumbnailSrc] = React.useState<string | null>(null);
+  const [neuroglancerUrl, setNeuroglancerUrl] = React.useState<string | null>(null);
+  const neuroglancerUrlPattern = "https://neuroglancer-demo.appspot.com/#!{%22layers%22:[{%22source%22:%22zarr://{URL}%22,%22name%22:%22OME-NGFF%22}]}";
```

Comment:
We are going to need to factor this out into something like an ENV variable. Future problem